PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -337,8 +379,27 @@ DECLARE_SOA_TABLE(DileptonsExtra, "AOD", "RTDILEPTONEXTRA", //!
                   reducedpair::Lz,
                   reducedpair::Lxy);
 
+DECLARE_SOA_TABLE(DileptonsAll, "AOD", "RTDILEPTONALL", //!
```

Comment:
Since this table contains MUON specific columns, i suggest to change the "Dileptons" in the name to "Dimuons".